### PR TITLE
Team7 sounds - added new sounds for the towers

### PR DESCRIPTION
**Pull request by karthikeya-v**
This PR is for the newly added sounds to the towers - TNT,stun,droid,fire and some test modifications.
look into the animation controllers and tests mainly as those are the only file modified and do run it and pls comment which sounds are too distracting or unsuitable.

### CODE REVIEW

File: source/core/src/main/com/csse3200/game/areas/ForestGameArea.java [hunk 1]
```diff
@@ -201,20 +198,22 @@ public class ForestGameArea extends GameArea {
...
-          "sounds/mobs/archerArrow.mp3"
-
+          "sounds/towers/Desert-Eagle-Far-Single-Gunshot.mp3",
+          "sounds/towers/5.56_single_shot.mp3",
+          "sounds/towers/explosion.mp3",
+          "sounds/towers/eco_tower_ping.mp3",
+          "sounds/towers/ar15_single_shot_far.mp3"
   };
-  private static final String BACKGROUND_MUSIC = "sounds/background/Sci-Fi1.ogg";
+  private static final String backgroundMusic = "sounds/background/Sci-Fi1.ogg";
```
Comment (The-AhmadAA) [inline source/core/src/main/com/csse3200/game/areas/ForestGameArea.java:207]: 'backGroundMusic' should be capitalised - was a code smell fix earlier today

File: source/core/src/main/com/csse3200/game/areas/ForestGameArea.java [hunk 2]
```diff
@@ -201,20 +198,22 @@ public class ForestGameArea extends GameArea {
...
 
-  private static final String[] forestMusic = {BACKGROUND_MUSIC};
+  private static final String[] forestMusic = {backgroundMusic};
```
Comment (The-AhmadAA) [inline source/core/src/main/com/csse3200/game/areas/ForestGameArea.java:209]: 'backGroundMusic' should be capitalised - was a code smell fix earlier today

File: source/core/src/main/com/csse3200/game/areas/ForestGameArea.java [hunk 3]
```diff
@@ -243,6 +242,37 @@ private void stopWaveTimer() {
     }
   }
 
+  /**
```
Comment (The-AhmadAA) [inline source/core/src/main/com/csse3200/game/areas/ForestGameArea.java:245]: Commented out code block should be removed - was a code smell fix earlier today

**Review by The-AhmadAA — CHANGES_REQUESTED** [review]
Please pull main into your branch and resolve merge conflicts before you try to merge -  there are quite a few code smell fixes that your merge would undo.

File: source/core/src/main/com/csse3200/game/areas/ForestGameArea.java (hunk 3 above)
Comment (karthikeya-v) [reply source/core/src/main/com/csse3200/game/areas/ForestGameArea.java:245]: got it

**Comment by karthikeya-v** [comment]
I merged main into it at 1:31 pm so anything after that is not there so guys if you just merged into main, leave a msg out here and I will do it again

**Review by The-AhmadAA — APPROVED** [review]

**Review by ThivanW — APPROVED** [review]
looks good to me! Just need to resolve the merge conflict

**Comment by Gaganx0** [comment]
Fix conflicts

**Review by The-AhmadAA — APPROVED** [review]

**Pull request merged**